### PR TITLE
Update read me for public access

### PR DESCRIPTION
**Pull request by robertblincoe**
## Pull request checklist

- [x] Add a descriptive message for this change to the PR
- [x] Run `black ./` locally
- [ ] Run `pylint <dashboard name>` locally and obtain a score of 10.00/10.00
- [ ] Run `python -u -m pytest --headless tests` locally and there are no failures
- [ ] Include screenshot for any visual changes
- [ ] Use cds_to_staging.py to push any new or updated data to the S3 bucket

### PR Description:
Copied important information from plotly docs for public access.

Due to making plotly docs private we've decided to update the template read.me to be useful to external users who want to copy the template. We considered that this may cause duplication as each repository based off the template will have it's own documentation. 
A positive is that the documentation will match the state of the template when it was copied. 
A negative is that future updates will need to be made across repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,73 @@ By default, GitHub  does not apply any branch protection policies to newly creat
...
+1.  Update the name field in environment.yml to the dashboard name
+1.  Create a new conda environment by typing `conda env create -f environment.yml` into the terminal and executing the command by pressing the Enter key.
+1.  Activate your conda environment with `conda activate <dashboard name> `
+1. Close VS Code. Open Anaconda Navigator, select "&lt; Dashboard name &gt;" for the 'Application on' drop down menu, then select "Launch" VS Code. Click the bin icon on the terminal toolbar to close the terminal. Click the plus icon on the terminal toolbar to launch a new terminal.
+1.  Install the [Microsoft Python][python_extension] extension for VS Code.
+1.  Follow the [instructions for configuring the Python interpreter][configure_python_interpreter].
+
+ **Note: If in DLUHC and get an error when creating conda environment check conda version. Type 'conda --version' in VS Code terminal. conda 4.5.12 needs to be upgraded to 4.10.3. Contact DAP-support@communities.gov.uk**
```
Comment (kateriley1) [inline README.md:71]: Do we want to include DAP supports email address if this repo is released to external users?

File: README.md [hunk 2]
```diff
@@ -86,3 +152,42 @@ More information on secrets can be found [here](https://docs.github.com/en/actio
...
+
+### Running the linter
+
+The linter checks for basic logic errors and bugs. Linting reports rule violations that must be corrected manually.  
+
+```bash
+pylint <Dashboard name>
+```
+
+###  Add reminder message to run formatter and linter before pushing to GitHub
```
Comment (adrianclay) [inline README.md:182]: Minor: Has anybody used these instructions?  I've not seen anybody use it while we've been pairing.

**Review by adrianclay — COMMENTED** [review]
My main comment is "Why copy this over?"

File: README.md (hunk 1 above)
Comment (tom-clarke-dluhc) [reply README.md:71]: Probably not - thanks for flagging!

File: README.md (hunk 2 above)
Comment (tom-clarke-dluhc) [reply README.md:182]: We tried to pick out the bits from the dashboard docs that would be useful for someone external. Happy to take feedback on sections that you don't think would be!

On the minor, I haven't but I'm about to set it up...

Comment (tom-clarke-dluhc) [reply README.md]: Didn't work - removed!

File: README.md (hunk 1 above)
Comment (tom-clarke-dluhc) [reply README.md]: The note has been removed

**Review by adrianclay — APPROVED** [review]

**Pull request merged**